src/remote/activitypub/audience.ts

```diff
@@ -88,6 +88,6 @@ function isPublic(id: string) {
 function isFollowers(id: string, actor: IRemoteUser) {
 	return [
 		`${actor.uri}/followers`,
-		// actor.followerUri,	// TODO
+		actor.followersUri,
```

Comment:
If it's null, I don't want to use it.

Reply:
good point!

to fix this i have redone `isFollowers` to compare `id` to either `followersUri` or the default

an account should only have one followers collection, so the default should only be as a backup